docs/contributor/reference/entity-lifecycle.md

```diff
@@ -28,6 +27,249 @@ There are two fundamental truths in this system:
 Beyond the above rules, lifecycle shifts occur at different times for different
 kinds of entities.
 
+(entity-creation)=
+## Entity creation
+
+This section describes the circumstances in which fundamental state entities
+are created, from the perspective of the CLI.
+
+### `juju deploy`
+
+The `juju deploy` command always creates services, may create relations, and
```

Comment:
The references to "services" here all need to be "application". This naming is vestigial.

Reply:
Fixed, thanks.